pydgraph/meta.py

```diff
@@ -14,4 +14,4 @@
 
 """Metadata about this package."""
 
-VERSION = '23.0.0rc1'
+VERSION = '23.0.0'
```

Comment:
@joshua-goldstein the plan is to have the same branching as our core dgraph project here. 

we should probably add a compatibility matrix in the landing page to show the client <> dgraph version.

Reply:
Actually that makes sense ^ .. whatever strategy we follow, it's better to add this to our developer docs for all projects.

Reply:
That makes sense. I think the strategy is what you guys decided. I am just saying that clients are too small projects that a linear history has to be good enough. It'd be very difficult for our customers to use the clients if the clients don't follow a linear history. We don't release branches here.

Reply:
I agree +1 ^ ...